Sources/Plasma/Apps/plPythonPack/main.cpp

```diff
@@ -426,61 +341,54 @@ void PackDirectory(const plFileName& dir, const plFileName& rootPath, const plFi
     }
 
     s.Close();
+    ST::printf(out, "\nPython Package written to {}\n", pakName);
 
     PythonInterface::finiPython();
 }
 
 void PrintUsage()
 {
-    fputs("Usage:\n", stdout);
-    fputs("plPythonPack [directory to pack...]\n", stdout);
-    fputs("NOTE: the directory to pack must have full system and plasma dirs and\n", stdout);
-    fputs("      must be a relative path to the current working directory\n", stdout);
+    ST::printf(out, "The Python Pack Utility\n");
+    ST::printf(out, "Usage:\n");
+    ST::printf(out, "plPythonPack [options] <directory to pack...>\n");
+    ST::printf(out, "NOTE: the directory to pack must have full system and plasma dirs.\n");
+    ST::printf(out, "\nAvailable options:\n");
+    ST::printf(out, "\t-q\tQuiet  - Only print errors\n");
+    ST::printf(out, "\t-h\tHelp   - Print this help\n");
 }
 
 int main(int argc, char *argv[])
 {
-    fputs("The Python Pack Utility\n", stdout);
-
-    plFileName baseWorkingDir = plFileSystem::GetCWD();
-
-    // are they asking for usage?
-    if (argc == 2)
-    {
-        std::string temp = argv[1];
-        temp = ToLowerCase(temp);
-        if ((temp == "?") || (temp == "-?") || (temp == "/?") || (temp == "-help") || (temp == "/help")
-            || (temp == "-h") || (temp == "/h"))
-        {
-            PrintUsage();
-            return -1;
+    // Parse arguments
+    ST::string packDir = ".";
+    for (int i = 1; i < argc; i++) {
+        ST::string arg = argv[i];
+
+        if (arg.starts_with("?") || arg.starts_with("-") || arg.starts_with("/")) {
```

Comment:
It might/is probably overkill, but worth remembering that we have [plCmdParser](https://github.com/H-uru/Plasma/blob/master/Sources/Plasma/CoreLib/plCmdParser.h) available. See [the tests](https://github.com/H-uru/Plasma/blob/master/Sources/Tests/CoreTests/test_plCmdParser.cpp) for example usage.

Reply:
::sigh:: I *thought* we had something, and I went looking and completely missed that.  Thanks, I should probably use it here. 🤦‍♂

Reply:
See comment by @dpogue 👅